extensions/app-search/README.md

```diff
@@ -0,0 +1,57 @@
+# App Search Server extension
+
+Adds a container for Elasticsearch App Search. 
+
+## Usage
+
+To include App Search:
+1. Update the root Docker Compose file to use the elasticsearch config in this extensions folder. **Note this requires you to use the basic license as of 7.6.0**
```

Comment:
I've been wondering why we need the users to switch their license type for App Search. As far as I know, `basic` allows _less_ features. Isn't it something we would want to let users decide once they reach the end of the trial, as described in the main README?

Reply:
Yeah, odd that giving *less* features is required but I just wanted to get it to work quickly (for self) and am not going to read too far into it until the time comes...